### PR TITLE
Dev 559

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: pages/guides/drop-ins/using-drop-in-components.mdx [hunk 1]
```diff
@@ -52,8 +52,10 @@ It's important to note that with the exception of a “Create a Customer” comp
 
 ```html
 <div class="customer">
-  <dwolla-business-vcr>
-    terms=”www.yourterms.com” privacy=”www.yourprivacy.com”>
+  <dwolla-business-vcr
+    terms="”www.yourterms.com”"
```
Comment (kellymore) [inline pages/guides/drop-ins/using-drop-in-components.mdx]: Double double-quotes

Comment (ShreyaThapa) [reply pages/guides/drop-ins/using-drop-in-components.mdx]: [This](https://github.com/Dwolla/dev-portal/pull/190/commits/d646e0a1bdc632db7859c8f0d6fd196430ade911) should fix it!

File: pages/guides/drop-ins/using-drop-in-components.mdx [hunk 2]
```diff
@@ -51,10 +51,7 @@ It's important to note that with the exception of a “Create a Customer” comp
 
 ```html
 <div class="customer">
-  <dwolla-business-vcr
-    terms="”www.yourterms.com”"
-    privacy="”www.yourprivacy.com”"
```
Comment (kellymore) [inline pages/guides/drop-ins/using-drop-in-components.mdx:57]: this one too

Comment (ShreyaThapa) [reply pages/guides/drop-ins/using-drop-in-components.mdx:57]: I think this fixed now, right?

**Pull request merged**